### DISCUSSION TITLE
How to show fields based on condition

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#310] [Q&A]
Hi
I am trying to modify one of my booking forms to only show a certain field based on the selection of another field.
Basically i want to tie a transponder number to a buggy

So i have a custom dropdown field REQ_FIELD{Entry Class} with selections for 1/8 ebuggy, 1/8 etruggy, 1/10 Shortcourse. Based on this selection i would like to show another custom field REQ_FIELD{Transponder  1/8 ebuggy}, REQ_FIELD{Transponder 1/8 etruggy}, REQ_FIELD{Transponder 1/10 Shortcourse}.

I am struggling to get my head round how to do this, have tried this so far but know it isnt correct

<td>#REQ_FIELD{Entry Class}</td>
<th scope="row">[eme_if='#_FIELD{Entry Class}'value='1/8 ebuggy']Transponder Number:</th>
<td>#_FIELD{Transponder_1/8 ebuggy}</td>

Has anyone with a better knowledge than me got any idea.

Thanks

**Comment by liedekef** [comment]
Take a look at this : https://www.e-dynamics.be/wordpress/registration-form-formatting/ , at the bottom you have a section called "Dynamic data" where EME can help you in this regard. Otherwise it needs to be done with own jquery code.
eme_if is based on existing data (so for example data from the event), not form data that is being filled out when the form shows.
